### DISCUSSION TITLE
In mmpretrain, Is "RandomResize" not supported?

### DISCUSSION THREAD

**Discussion by yellowjs0304** [open-mmlab/mmclassification#1538] [Q&A]
Hi, I want to train/evaluate the model(efficientNetv2) not to crop the image.
It looks like the config file "train-pipeline, test-pipeline" only support "RandomResizedCrop" .

When I modify the line to use "randomresize", there's some error like below. 

What Can i do for this problem.
![image](https://user-images.githubusercontent.com/30281608/235561973-c2396c65-abc8-4061-be07-cc5a7bc2c8db.png)
![image](https://user-images.githubusercontent.com/30281608/235562199-7f5c61cc-6f93-46bc-8602-bb4d3f65d897.png)

**Comment by Ezra-Yu** [comment]
if the scale is a tuple? the `ratio_range` must be a `Tuple[float, float]`
